src/pages/Bridge/__tests__/index.spec.tsx

```diff
@@ -140,6 +142,12 @@ describe('Bridge', () => {
   });
 
   it('updates input value correctly when clicking on max button', async () => {
+    const fakeBalanceMantissa = new BigNumber('10000000000000000000');
+    (useGetBalanceOf as Vi.Mock).mockImplementation(() => ({
+      data: {
+        balanceMantissa: fakeBalanceMantissa,
+      },
+    }));
     const { getByText, getByTestId } = renderComponent(<Bridge />, {
       chainId: ChainId.BSC_TESTNET,
     });
```

Comment:
We'll need to add more tests (for example to check that the mutation is called properly when submitting the form)